scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -244,8 +234,8 @@ task SelectVariants {
   command <<<
     gatk --java-options "-Xms~{command_mem}m -Xmx~{max_heap}m" \
       SelectVariants \
+        -L ~{sep=' -L ' chromosomes} \
```

Comment:
technically this can take in any interval list, right? I think L can take strings or a file, right? Is that something we want to expand to in the future?